Board.java

```diff
@@ -0,0 +1,634 @@
+package test;
+import java.util.ArrayList;
+
+public class Board {
+    public class BoardSquare {
+        private Tile tile;
```

Comment:
private final

Reply:
i'm changing the tile variable later in the class so final is not possible here

Reply:
i'm changing the tile variable later in the class so final is not possible here